app/main.py

```diff
@@ -1,6 +1,29 @@
+import os
+import requests
+
+
+BASE_URL = "http://api.weatherapi.com/v1/current.json"
+Q_FILTER = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    url = f"{BASE_URL}"
+    response = requests.get(
+        url,
+        params={"q": Q_FILTER, "key": {os.getenv("API_KEY")}},
+    )
+
+    if response.status_code == 200:
+        result = response.json()
+        condition = result["current"]["condition"]["text"]
+        local_time = result["location"]["localtime"]
+        weather = (f"Weather: "
+                   f"{result['current']['temp_c']} Celsius, {condition}")
+        print(f"Paris/France, {local_time}, {weather}")
+    elif response.status_code == 403:
+        print("Your API_KEY is invalid or not provided.")
+    else:
```

Comment:
It would be better to not consider default success response in case `status_code` is different from `403`, since there could be other codes from 4xx or 5xx groups. Explicitly check for a success response before retrieving data further on. For better error handling you could include additional checks for required data presence in response before retrieving them from attributes